packages/fetch-api/src/normalize/gif.ts

```diff
@@ -29,10 +29,17 @@ type Tag = { text: string }
 // tags sometimes are objects that have a text prop, sometimes they're strings
 const getTag = (tag: Tag | string) => (typeof tag === 'string' ? (tag as string) : (tag as Tag).text)
 
-const normalize = (gif: any) => {
+const normalize = (gif: any, responseId: string = '') => {
     const newGif: IGif = { ...gif }
     newGif.id = String(newGif.id)
     newGif.tags = (newGif.tags || []).map(getTag)
+    if (!newGif.bottle_data) {
+        newGif.bottle_data = {} as any
+    }
+    if (responseId) {
+        newGif.response_id = responseId
+    }
+    newGif.response_id = responseId
```

Comment:
@nicksantan looks like you do this twice, I don't think we need to wrap it in the `if (responseId)` check